packages/ag-grid-theme/src/__tests__/__e2e__/AgGridTheme.cy.tsx

```diff
@@ -0,0 +1,456 @@
+import { composeStories } from "@storybook/react";
+import * as agGridStories from "@stories/ag-grid-theme.stories";
+
+const {
+  BasicGrid,
+  CheckboxSelection,
+  Coloration,
+  ColumnGroup,
+  ColumnSpanning,
+  ContextMenu,
+  CustomFilter,
+  DragRowOrder,
+  Icons,
+  FloatingFilter,
+  HDCompact,
+  HDCompactDark,
+  InfiniteScroll,
+  LoadingOverlay,
+  MasterDetail,
+  MasterDetailDark,
+  NoDataOverlay,
+  Pagination,
+  ParentChildRows,
+  RowGrouping,
+  RowGroupPanel,
+  PinnedRows,
+  StatusBar,
+  StatusBarDark,
+  VariantSecondary,
+  VariantSecondaryDark,
+  VariantZebra,
+  VariantZebraDark,
+  WrappedHeader,
+} = composeStories(agGridStories);
+
+describe("Given Ag Grid Theme", () => {
+  describe("WHEN the Default story is mounted", () => {
+    it("THEN should match screenshot", () => {
+      cy.mount(<BasicGrid />);
+      cy.wait(500);
+      cy.matchImage({
+        maxDiffThreshold: 0.1,
```

Comment:
`pluginVisualRegressionDiffConfig` setting in the config doesn't work ?

Reply:
seems not to work

Reply:
Looking at the [doc](https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff#configuration) on the prefix part, should it be `pluginVisualRegressionMaxDiffThreshold`?